microproxy/viewer/format/formatter.py

```diff
@@ -66,14 +66,12 @@ def format_body(self, body):
         return "\n".join(pretty_string)
 
     def _try_parse_value(self, value):
-        if not value:
-            return value
         try:
             if value[0] in self.JSON_SUSPECT_PREFIX:
-                return "\n" + self.json_formatter.format_body(value)
+                value = "\n" + self.json_formatter.format_body(value)
             if value[0] in self.XML_SUSPECT_PREFIX:
-                return "\n" + self.xml_formatter.format_body(value)
-        except:
+                value = "\n" + self.xml_formatter.format_body(value)
+        finally:
             return value
```

Comment:
Sorry for bring this bug that I forgot to return value for default situation.
Finally is not used for return value.

If you prefer to design it as a single exit point function,
then I think you should try like this one
```python
try:
        ......
except:
        pass

return value
```

And the two if should update second one into elif,
because this will be a confused case that is the message will be format as xml after it format as json?